src/commands/create/index.ts

```diff
@@ -185,28 +189,12 @@ export default class Create extends Command {
         });
     }
 
-    private async _setDefaultAppManifestInterfaceConfig(interfaces: string[]) {
-        if (this.appManifestInterfaces.interfaces.length > 0) {
+    private async _promptMissingInterfaceConfig(interfaces: string[]): Promise<void> {
```

Comment:
The argument name should convey what it really is. `interfaces` is very generic ;-)
```suggestion
    private async _promptMissingInterfaceConfig(inputInterfaces: string[]): Promise<void> {
```